### ISSUE TITLE
Random output - Attention mask and the pad token id were not set

### ISSUE THREAD

**Issue by akjoshi** [edwko/OuteTTS#18]
Input text is ignored and random audio output is generated by random speaker.

How to fix below?

The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:None for open-end generation.
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results

Using the same code as in blog: https://www.outeai.com/blog/outetts-0.1-350m
``
```
from outetts.v0_1.interface import InterfaceHF, InterfaceGGUF

# Initialize the interface with the Hugging Face model
interface = InterfaceHF("OuteAI/OuteTTS-0.1-350M")

# Or initialize the interface with a GGUF model
# interface = InterfaceGGUF("path/to/model.gguf")

# Generate TTS output
# Without a speaker reference, the model generates speech with random speaker characteristics
output = interface.generate(
    text="Hello, am I working?",
    temperature=0.1,
    repetition_penalty=1.1,
    max_lenght=4096
)

# Play the generated audio
output.play()

# Save the generated audio to a file
output.save("output.wav")
```
``

**Comment by edwko** [comment]
You can ignore this warning for the interface. It’s informing you that the tokenizer doesn't have a pad token set.
